### ISSUE TITLE
Create event clock is stupid

### ISSUE THREAD

**Issue by klstilling** [connors511/dinto-issues#81]
It would be very nice if the clock had a sane default state, like the current time, or the time from the current 'at' state so you don't have to actively set it when you are in a hurry.

Also, it would be nice if it wrapped around, so that scrolling beyond 23 would go to 0->1->2...
The same goes for the minutes wheel.

**Comment by connors511** [comment]
I totally agree, but the current date/time picker is the best library I've found and I haven't had the time to create something from scratch.

**Comment by klstilling** [comment]
@connors511 after spending half an hour on searches, I must admit that finding a decent time picker library is hard. Roll your own ;)

**Comment by connors511** [comment]
For reference @klstilling linked https://puranjayjain.github.io/md-date-time-picker/ and https://domon.me/vue-clock-picker/ as possible inspiration for DYI.
Will revisit this later.

**Comment by connors511** [comment]
Related to #29
